src/frontend/src/views/Data/components/TreeActionMenu/components/MoreActionsMenu/style.ts

```diff
@@ -26,3 +36,16 @@ export const StyledTrashIcon = styled(TrashIcon)`
     `;
   }}
 `;
+
+export const StyledEditIcon = styled(EditIcon)`
+  fill: black;
```

Comment:
I'm fine with this being black, but considering the above, it's interesting to think how our fill colors are ever so slightly different than our text colors. This might be something we want to bring up to Janeece, maybe talk to SDS team about.

Reply:
Janeece already design approved this, and `black` is listed in the product spec, so I think we're ok here